model/Lite/Lite.md

```diff
@@ -49,3 +47,19 @@ the following has to hold:
    `/Core/Relationship` of type `hasDeclaredLicense` having that element as its
    `from` property and an `/SimpleLicensing/AnyLicenseInfo` as its `to`
    property.
+
+Finally, for a `/Core/Agent` to be conformant with this profile, the following has to hold:
+
+1. The mincount for `name` is 1
+
+
+
+<!--
```

Comment:
What does this commented out block mean?

Reply:
I'm waiting to update the Core classes that they should have at least one element and then I can delete this.

If we don't update the classes, I'll add it here, because SPDX-Lite requires non-empty SpdxDocuments and SBOMs.

:-)

Reply:
Got it, Thanks.
This change is also looks great.